src/Cache/SchemaCache.php

```diff
@@ -6,6 +6,8 @@
 
 use Psr\SimpleCache\CacheInterface;
 use Yiisoft\Cache\CacheKeyNormalizer;
+use Yiisoft\Cache\Dependency\Dependency;
+use Yiisoft\Cache\Dependency\TagDependency;
 
 final class SchemaCache
```

Comment:
Need docs overall about this class and its methods. Some aren't clear enough without docs.

Reply:
done,